extensions/BMO/template/en/default/hook/flag/type_comment-form.html.tmpl

```diff
@@ -225,3 +225,97 @@
     </table>
   </section>
 </template>
+
+<template class="approval-request" data-flags="data-review">
+  <section>
+    <header>
+      <h3>Data review request</h3>
+    </header>
+    <table>
+      <tr>
+        <th id="_ar_sec_i1_label">What questions will you answer with this data?</th>
+        <td><textarea aria-labelledby="_ar_sec_i1_label"></textarea></td>
+      </tr>
+      <tr>
+        <th id="_ar_sec_i2_label">Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?</th>
+        <td><textarea aria-labelledby="_ar_sec_i2_label">
+          Examples:
+* Establish baselines or measure changes in product or platform quality or performance.
+* Provide information essential for advancing a business objective such as supporting OKRs.
+* Determine whether a product or platform change has an effect on user or browser behavior.
+        </textarea></td>
+      </tr>
+      <tr>
+        <th id="_ar_sec_i3_label">What alternative methods did you consider to answer these questions? Why were they not sufficient?</th>
+        <td><textarea aria-labelledby="_ar_sec_i3_label"></textarea></td>
+      </tr>
+      <tr>
+        <th id="_ar_sec_i4_label">Can current instrumentation answer these questions?</th>
+        <td><textarea aria-labelledby="_ar_sec_i4_label"></textarea></td>
+      </tr>
+      <tr>
+        <th id="_ar_sec_i5_label">List all proposed measurements and indicate the category of data collection for each measurement, using the
+          <a href="https://wiki.mozilla.org/Data_Collection">Firefox data collection categories</a> found on the Mozilla wiki:</th>
+        <td>
+          <textarea aria-labelledby="_ar_sec_i5_label">
+For each measurements, please provide:
+* Measurement Description
+* Data Collection Category
+* Tracking Bug #
```

Comment:
`* Tracking [% terms.Bug %]`

Any mention of bug, Bug, Bugzilla, etc. has a matching [% terms.Bug %] variable that must be used. This is the reason for the sanity test failure.